test/functional/stats_controller_test.rb

```diff
@@ -52,14 +52,14 @@ class StatsControllerTest < ActionController::TestCase
 
   context "on GET to index with multiple gems" do
     setup do
+      create(:gem_download, count: 0)
       rg1 = create(:rubygem, downloads: 10, number: "1")
       rg2 = create(:rubygem, downloads: 20, number: "1")
       rg3 = create(:rubygem, downloads: 30, number: "1")
       n = 10
-      [rg1, rg2, rg3].each do |rg|
-        rg.versions.last.gem_download.update(count: n)
```

Comment:
We do it in bulk_update so that we don't have to update the row once for each version, we can do one update for the batch.